src/rapids_pre_commit_hooks/copyright.py

```diff
@@ -125,14 +125,16 @@ def get_target_branch(repo, target_branch_arg=None):
       used. This allows GitHub Actions inside ``copy-pr-bot`` to easily use this tool.
     * If the configuration option ``rapidsai.baseBranch`` is defined, that branch is
       used. This allows users to locally set a base branch on a long-term basis.
```

Comment:
Is `rapidsai.baseBranch` a git configuration option? I struggled to find how to set this earlier. Clarifying the docstring would help a lot.